### PR TITLE
bump fuse online to 1.7.26

### PR DESCRIPTION
**Pull request by aidenkeating**
@maleck13 @StevenTobin Does this look correct to you?

Jira: [INTLY-3953](https://issues.jboss.org/browse/INTLY-3953)

### CODE REVIEW

**Comment by aidenkeating** [comment]
verified on OSD and using heimdall to ensure images are up to date.

it's worth noting that `fuse7-tech-preview/data-virtualization-server-rhel7` is marked as not up to date by heimdall. however, the version does match what they have in their release files.

```
| komodo-server       | fuse7-tech-preview/data-virtualization-server-rhel7 | true         | 1.4-15  | true               | 1.4-15               | 1.4-15.1567588155    | 1.4          | false              | false                       |             0 |              1 |            40 |
```

File: playbooks/upgrades/upgrade.yaml [hunk 1]
```diff
@@ -18,60 +18,16 @@
         tasks_from: upgrade
       vars:
         from_versions:
-          - "release-1.5.1"
```
Comment (merlante) [inline playbooks/upgrades/upgrade.yaml]: Given this change didn't make 1.5.2, I'd expect this to change.

**Comment by jessesarn** [comment]
@aidenkeating This can be closed because its superseded by https://github.com/integr8ly/installation/pull/1133, correct?

**Comment by aidenkeating** [comment]
@jessesarn There ended up being some confusion here. So this can be closed off, it's superseded by:

https://github.com/integr8ly/installation/pull/1153 for v1.5 (not yet verified on OSD)
https://github.com/integr8ly/installation/pull/1133 for master (verified on OSD, logs in Jira ticket)

**Pull request closed without merging**